src/brouwers/kitreviews/models.py

```diff
@@ -1,17 +1,20 @@
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.core.urlresolvers import reverse
+from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
+from djchoices import DjangoChoices, ChoiceItem
 
 from brouwers.albums.models import Album
 from brouwers.general.utils import get_username
 
-
 RATING_BASE = 100  # store ratings relative to 100
 RATING_DISPLAY_BASE = 5
 DEFAULT_RATING = 50
 DEFAULT_DIFFICULTY = 3
```

Comment:
Yeah, I was thinking the same. The DEFAULT_RATING is used  afaik, but there rest don't seem necessary anymore.